tensorboard/webapp/plugins/npmi/views/annotations_list/annotation/annotation_component.ts

```diff
@@ -176,7 +176,7 @@ export class AnnotationComponent implements AfterViewInit, OnChanges {
 
     this.yScale
       .rangeRound([0, this.chartHeight - this.runHeight])
-      .domain(this.runs.map((d: string) => d));
```

Comment:
I'm curious why this changes.

Before, we are creating a copy of `this.runs` before passing it to D3's scale.domain().  Are we fixing cases where making copies was unnecessary?  If so, should we also update the `domain([...this.runs])` callsites below in this file?

Reply:
The map function did not do anything and yes, there does not need to be a copy, that is why I removed it.

Agreed, the other copies also should not be made. Removed them.